src/renderer/components/+workloads-pods/details/volumes/variants/projected.tsx

```diff
@@ -10,12 +10,14 @@ import type { VolumeVariantComponent } from "../variant-helpers";
 export const Projected: VolumeVariantComponent<"projected"> = (
   ({ variant: { sources, defaultMode }}) => (
     <>
-      <DrawerItem name="Default Mount Mode">
-        {`0o${defaultMode.toString(8)}`}
-      </DrawerItem>
+      {typeof defaultMode === "number" && (
+        <DrawerItem name="Default Mount Mode">
+          {`0o${defaultMode.toString(8)}`}
+        </DrawerItem>
+      )}
       <DrawerItem name="Sources">
```

Comment:
Should we show this at all if `sources` is null/undefined? (as long as we're consistent with what is done elsewhere)

Reply:
I would say yes, but we can always change it in the future.